Release 3.11 update (2nd attempt) (#294)
sehlen-bsi/botan-docs: docs/cryptodoc/src/05_03_ecc.rst

> silent*. However, it is used for only operations that don't handle secret values

Reviewer: ```suggestion
silent*. However, it is used for only operations that do not handle secret values
```

   **Steps:**

>    1. During the initialization the signature operation, sample a random blinding scalar :math:`1 \leq b_i < n` from ``rng``

Reviewer: ```suggestion
   1. During the initialization of the signature operation, sample a random blinding scalar :math:`1 \leq b_i < n` from ``rng``
```

   2. :math:`e = H` and truncate :math:`e` to be at most bitlength(:math:`n`) bits long.

   3. Generate parameter :math:`k` as a random number :math:`0 < k < n` using
      the algorithm described in Section :ref:`pubkey_param/rng` or as HMAC_DRBG
      output [RFC6979]_. If Botan is compiled with the module RFC6979 the
      HMAC_DRBG is used, otherwise :math:`k` is sampled from the passed random
      number generator ``rng``.

   4. :math:`r = R_x \bmod n` where :math:`R_x` is the x-coordinate of the affine point
      :math:`R_{x,y} = k \cdot G` (see :ref:`pubkey/ecc/scalar_mul`). The computation
      of :math:`k \cdot G` is done with the use of scalar blinding.

   5. Compute :math:`k'^{-1} = (b_i \cdot k)^{-1}` using the blinding
      value :math:`b_i`. The inversion either uses Fermat's little theorem (:math:`a^{-1} = a^{p-2} \bmod n`) or a
      curve-specific addition chain if available. In the former case, the computation is realized with a variable time scalar exponentiation.

   6. Compute :math:`u=(d \cdot b_i) \cdot r + e \cdot b_i`. The braces indicate the order of evaluation.

>    7. Compute :math:`s = k'^{-1} \cdot u` which amounts to :math:`(b_i \cdot k )^{-1} \cdot (d \cdot r + e) \cdot b_i = k^{-1} \cdot (d \cdot r + m)`.

Reviewer: ```suggestion
   7. Compute :math:`s = k'^{-1} \cdot u` which amounts to :math:`(b_i \cdot k )^{-1} \cdot (d \cdot r + e) \cdot b_i = k^{-1} \cdot (d \cdot r + e)`.
```

   8. Square the blinding value :math:`b_i` to

>       obtain new a blinding value :math:`b_{i+1}` which replaces :math:`b_i` in

Reviewer: ```suggestion
      obtain a new blinding value :math:`b_{i+1}` which replaces :math:`b_i` in
```
